### PR TITLE
Update Prototype Kit accessibility statement and CONTRIBUTING file

### PR DESCRIPTION
**Pull request by kellylee-gds**

### CODE REVIEW

File: docs/v12/documentation/accessibility.md [hunk 1]
```diff
@@ -66,5 +64,5 @@ When we publish new content, we’ll continue to make sure that it meets accessi
 
 ### Preparation of this accessibility statement
 
-This statement was prepared on 5 November 2021. It was last reviewed on 6 September 2022.
+This statement was prepared on 5 November 2021. It was last reviewed on 11 January 2024.
```
Comment (36degrees) [inline docs/v12/documentation/accessibility.md]: I'm not 100% sure we should update this date – although we're revising the statement to update the contact details, we haven't _reviewed_ it (i.e. checked that it's still an accurate representation of the accessibility of the docs site) in any way.

@selfthinker do you have any thoughts on what we should do here?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,14 +1,15 @@
...
+
+[There is only minimal support in place](https://github.com/alphagov/govuk-prototype-kit/issues/2389) so we cannot work on contributions at this time.
 
 ## Contributing
 
 If you’ve got an idea or suggestion you can:
 
-* email [govuk-prototype-kit-support@digital.cabinet-office.gov.uk](mailto:govuk-prototype-kit-support@digital.cabinet-office.gov.uk)
-* [get in touch on developer Slack channel](https://ukgovernmentdigital.slack.com/archives/C0E1063DW)
+* [share with other users on the Slack channel](https://ukgovernmentdigital.slack.com/archives/C0E1063DW)
```
Comment (36degrees) [inline CONTRIBUTING.md]: This links to `#prototype-kit-dev`, which isn't the 'main' Slack channel for the Prototype Kit.

Suggest we either using a more descriptive link so users know where they're going, or pointing users to the main `#prototype-kit` channel?

File: docs/v12/documentation/accessibility.md (hunk 1 above)
Comment (kellylee-gds) [reply docs/v12/documentation/accessibility.md]: @36degrees Anika may not be able to review this as she's focused on GOV.UK audit work currently. I agree with your note though so let's revert it back to the original date.

**Comment by 36degrees** [comment]
I've [force-pushed](https://github.com/alphagov/govuk-prototype-kit-docs/compare/c5a0aef8789296fbbbe813c7789b326462a26604..5f88ca2f7b7df5c1b8c9ceb4de1fcdb59985a227) to tidy up the commit history so we now have 3 commits:

- 1b3a4d7 updates the accessibility statement for v13 docs
- fb77fa1 updates the accessibility statement for v12 docs
- 5f88ca2 updates the contribution docs

Only the commit structure changed – the diff remains the same as it did before.

**Review by 36degrees — APPROVED** [review]

**Pull request merged**